app/templates/index.html

```diff
@@ -0,0 +1,146 @@
+             685.539C250.107 682.031 248.436 678.773 246.087 676.031C243.737 673.288 240.775 671.137 237.439 669.752C234.104 668.367 
+             230.489 667.787 226.888 668.059C223.287 668.331 219.8 669.447 216.71 671.317C213.621 673.187 211.015 675.758 209.103 
+             678.823C207.192 681.887 206.03 685.358 205.71 688.956L215.568 689.833C215.75 687.783 216.412 685.805 217.501 684.06C218.59 
+             682.314 220.075 680.849 221.835 679.784C223.595 678.718 225.581 678.082 227.633 677.928C229.685 677.773 231.744 678.103 233.644 
+             678.892C235.544 679.681 237.232 680.907 238.571 682.469C239.909 684.032 240.861 685.887 241.349 687.886C241.837 689.885 241.847 
+             691.971 241.379 693.974C240.911 695.978 239.977 697.843 238.654 699.419L246.234 705.783Z" fill="#EBEBFF"/>
+            <rect x="1112" y="102" width="204" height="47" transform="rotate(-30 1112 102)" fill="#EBEBFF"/>
+            <rect x="697.44" y="631" width="161.407" height="28.875" transform="rotate(23.34 697.44 631)" fill="#EBEBFF"/>
+            <rect width="56.0124" height="10.0203" transform="matrix(0.977401 -0.211395 -0.211395 -0.977401 659.253 142.572)" fill="#6070FF"/>
+            <rect x="0.601562" y="99.9232" width="108.669" height="19.4403" transform="rotate(-46.1415 0.601562 99.9232)" fill="#EBEBFF"/>
+            <circle cx="828.5" cy="662.5" r="23.5" fill="#6070FF"/>
+            <circle cx="685.5" cy="149.5" r="11.5" fill="#EBEBFF"/>
+            <circle cx="1177.5" cy="562.5" r="37" fill="white" stroke="#EBEBFF"/>
+            <circle r="19" transform="matrix(1 0 0 -1 950.5 53.5)" fill="white" stroke="#EBEBFF"/>
+            <circle cx="352.5" cy="53.5" r="7" fill="white" stroke="#EBEBFF"/>
+            <circle r="11" transform="matrix(1 0 0 -1 50.5 404.5)" fill="white" stroke="#EBEBFF"/>
+            <path d="M46.9842 73.0741C42.7936 82.8892 42.6736 93.967 46.6508 103.87C50.6279 113.774 
+            58.3763 121.692 68.1914 125.883C78.0065 130.073 89.0843 130.193 98.9878 126.216C108.891 
+            122.239 116.809 114.49 121 104.675L46.9842 73.0741Z" fill="#6070FF"/>
+            <path d="M221.155 650C220.821 654.881 222.439 659.695 225.654 663.383C228.87 667.071 233.418 
+            669.331 238.299 669.666C243.181 670 247.995 668.381 251.683 665.166C255.371 661.951 257.63 
+            657.403 257.965 652.521L221.155 650Z" fill="#6070FF"/>
+        </svg> 
+        
+        <!--Menu-->
+        <nav>
+            <div class="top-left">
+                <div class="menu"></div>
+                <div class="menu"></div>
+                <div class="menu"></div>
+            </div>
+            <div class="top-right">
+                <h2>My Logo</h2>
+            </div>
+        </nav>
+
+        <!-- Headline-->
+        <div class="centered">
+            <div class="primary-text">
+                <p>I’m Kennedy <br> Glad to see you!</p>
```

Comment:
- [ ] please its preferable to use the `h1 `tag here since it is a heading and ensure it is bold and has the figma size